src/components/Tables/OrderProcessorTable/index.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Paper,
+  Stack,
+  Table,
+  TableBody,
+  TableContainer,
+  TableFooter,
+  TableHead,
+  TablePagination,
+  TableRow,
+} from '@mui/material';
+import TimeAgo from 'javascript-time-ago';
+import en from 'javascript-time-ago/locale/en';
+import { useState } from 'react';
+
+import { getBalanceString } from '@/utils/functions';
+
+import { Address, Link } from '@/components/Elements';
+
+import { SUSBCAN_CORETIME_URL } from '@/consts';
+import { useCoretimeApi } from '@/contexts/apis';
+import { useNetwork } from '@/contexts/network';
+import { OrderItem } from '@/models';
+
+import { StyledTableCell, StyledTableRow } from '../common';
+
+interface OrderProcessorTableProps {
+  data: OrderItem[];
+}
+
+export const OrderProcessorTable = ({ data }: OrderProcessorTableProps) => {
+  TimeAgo.addLocale(en);
+  // Create formatter (English).
+  const timeAgo = new TimeAgo('en-US');
+
+  const { network } = useNetwork();
+  const {
+    state: { decimals },
+  } = useCoretimeApi();
+
+  // table pagination
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (
+    _event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  return (
+    <Stack direction='column' gap='1em'>
+      <TableContainer component={Paper} sx={{ height: '32rem' }}>
+        <Table stickyHeader>
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>Order Id</StyledTableCell>
+              <StyledTableCell>Extrinsic Id</StyledTableCell>
+              <StyledTableCell>Who</StyledTableCell>
+              <StyledTableCell>Reward</StyledTableCell>
+              <StyledTableCell>Timestamp</StyledTableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {(rowsPerPage > 0
+              ? data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              : data
+            ).map(
+              ({ orderId, extrinsicId, account, reward, timestamp }, index) => (
+                <StyledTableRow key={index}>
+                  <StyledTableCell align='center'>{orderId}</StyledTableCell>
```

Comment:
This should be clickable. When clicked, it should open a modal that includes the order data (`<OrderCard/>`).

Reply:
Okay, got it.